src/absil/ilread.fs

```diff
@@ -3942,7 +3945,13 @@ let rec genOpenBinaryReader infile is opts =
 #if NO_PDB_READER
     let pdb = None
 #else
-    let pdb = if runningOnMono then None else getPdbReader opts infile
+    let pdb = 
+#if FX_RUNNING_ON_MONO
+        if runningOnMono then 
+            None 
+        else 
+#endif
+            getPdbReader opts infile
```

Comment:
With 


    #else 
    let  runningOnMono=false
    #else

this kind of change won't be needed